groups/nts/ntsf/ntsf_system.h

```diff
@@ -1563,6 +1563,11 @@ struct System {
     /// function when possible.
     static bool supportsTransport(ntsa::Transport::Value transport);
 
+    /// Load into the specified `result` names of tcp congestion control
+    /// algorithms available for the process. Return the error.
+    static ntsa::Error getTcpCongestionControlAlgorithms(
```

Comment:
We have a small precedent for this pattern when configuring the back-end "driver". See `ntcf::System::loadDriverSupport()`. I think we should extended this pattern for the congestion control algorithms: Rename `getTcpCongestionControlAlgorithms()` to `loadCongestionControlAlgorithmSupport()` and add`testCongestionControlAlgorithmSupport()`.

Reply:
Done. I just prefered to keep "Tcp" part of the function name. Also,  think it's important have comparison case sensitive in the "test" function.